codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -12,6 +12,8 @@ Expressions evaluating SKv2 Config use [[ "[[" ]] and [[ "]]" ]]
   [[- $workloadKind = "DaemonSet" ]]
   [[- end -]]
 [[- $operatorVar := (lower_camel $operator.Name) -]]
+[[- $extraPodLabels := $operator.Deployment.Labels -]]
+[[- $extraPodAnnotations := $operator.Deployment.Annotations -]]
```

Comment:
Should we put these labels on the deployment as well? I'm not sure about the conventions around Pod labels vs. Deployment labels, usually I see the same set of labels on both

@ilackarms any thoughts?

Reply:
I'll add a `extraPodLabels` and `extraDeploymentLabels` field (same for annotations). This allows us to be flexible with which labels/annotations go where.